plugins/env/app/views/samson_env/_environment_variables_fields.html.erb

```diff
@@ -0,0 +1,33 @@
+<% environment_variables << EnvironmentVariable.new %>
+<%= form.fields_for :environment_variables, environment_variables do |fields| %>
+  <div class="form-group">
+    <div class="col-lg-3">
+      <%= fields.text_field :name, class: "form-control", placeholder: "Name" %>
+    </div>
+
+    <div class="col-lg-5">
+      <%# using a text area so users can resize them with browser controls %>
+      <%= fields.text_area :value, class: "form-control", placeholder: "Value", rows: 1 %>
+    </div>
+
+    <% if DeployGroup.enabled? %>
```

Comment:
the scope should not be shown on the deploy page since it's not being used, right ? ... so `&& !@deploy`

Reply:
true that! I don't have the DeployGroup enabled, so I didn't see that... will add that condition

Reply:
although, will use `@deploy.nil?`

Reply:
Although, this is not gonna work because we are not passing the `scopes` variable here from the parent view, and we do need that `scope` var to generate the actual env variables list for the project.

So I've decided to use the presence of that variable `scope`, as the condition to show or hide the `scope_type_and_id` param

Also, I did enable / disabled the feature with `DEPLOY_GROUP_FEATURE=true` and ensure everything worked fine.